<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="vilens_graph_topic"     default="/vilens_slam/pose_graph"/>
  <arg name="vilens_state_topic"     default="/vilens/state_propagated"/>

  <!-- Optional topics -->
  <arg name="operator_twist_topic"   default="/motion_reference/command_twist"/>
  <arg name="tf_frames"              default="/state_estimator/anymal_state"/>


  <!-- Launch mission_analysis_node -->
  <node pkg="digiforest_analysis_ros" type="mission_analysis_node.py" name="anymal_digiforest_mission_analysis" output="screen" >
    <param name="vilens_graph_topic"     value="$(arg vilens_graph_topic)" />
    <param name="vilens_state_topic"     value="$(arg vilens_state_topic)" />

    <param name="operator_twist_topic"   value="$(arg operator_twist_topic)" />
    <param name="tf_frames"              value="$(arg tf_frames)" />
  </node>

</launch>